contracts/ComptrollerInterface.sol

```diff
@@ -75,6 +75,13 @@ abstract contract ComptrollerInterface {
     ) external view virtual returns (uint256, uint256);
 
     function getAllMarkets() external view virtual returns (VToken[] memory);
+
+    /**
+     * @dev This empty reserved space is put in place to allow future versions to add new
+     * variables without shifting down storage in the inheritance chain.
+     * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+     */
+    uint256[50] private __gap;
```

Comment:
Shouldn't we only include this gap in the Storage interface? I can see we have this gap added on both of them?

Reply:
If it's an interface, maybe we should explicitly mark it as `interface` rather than `abstract contract`?

Reply:
Done